testing/tests/DevExpress.core/domComponent.tests.js

```diff
@@ -1,931 +1,957 @@
+                return this._createComponent(element, name, config);
+            }
+        }));
 
-    var firstInstance = $firstElement.TestComponent({ rtlEnabled: true }).TestComponent("instance"),
-        secondInstance = firstInstance.createComponent($secondElement, "TestComponent", {});
+        const firstInstance = $firstElement.TestComponent({ rtlEnabled: true }).TestComponent("instance");
+        const secondInstance = firstInstance.createComponent($secondElement, "TestComponent", {});
 
-    assert.ok(secondInstance.option("rtlEnabled"), true);
-});
+        assert.ok(secondInstance.option("rtlEnabled"), true);
+    });
 
-QUnit.test("'templatesRenderAsynchronously' option is passed to createComponent", function(assert) {
-    var $firstElement = $("#component"),
-        $secondElement = $("#anotherComponent");
+    QUnit.test("'templatesRenderAsynchronously' option is passed to createComponent", (assert) => {
+        const $firstElement = $("#component");
+        const $secondElement = $("#anotherComponent");
 
-    registerComponent("TestComponent", this.TestComponent.inherit({
-        createComponent: function(element, name, config) {
-            return this._createComponent(element, name, config);
-        }
-    })
-    );
+        registerComponent("TestComponent", this.TestComponent.inherit({
+            createComponent(element, name, config) {
+                return this._createComponent(element, name, config);
+            }
+        }));
 
-    var firstInstance = $firstElement.TestComponent({ templatesRenderAsynchronously: true }).TestComponent("instance"),
-        secondInstance = firstInstance.createComponent($secondElement, "TestComponent", {});
+        const firstInstance = $firstElement.TestComponent({ templatesRenderAsynchronously: true }).TestComponent("instance");
+        const secondInstance = firstInstance.createComponent($secondElement, "TestComponent", {});
 
-    assert.ok(secondInstance.option("templatesRenderAsynchronously"), true);
-});
+        assert.ok(secondInstance.option("templatesRenderAsynchronously"), true);
+    });
 
-QUnit.test("option 'rtl'", function(assert) {
-    var $element = $("#component").TestComponent(),
-        instance = $element.TestComponent("instance");
+    QUnit.test("it should be possible to define aliases for integration templates", (assert) => {
+        const instance = new DomComponentWithTemplate("#component", {
+            template: "name1",
+            integrationOptions: {
+                templates: {
+                    name1: {
+                        render() {
+                            return "Markup for name 1";
+                        }
+                    },
+                    name2: {
+                        render() {
+                            return "Markup for name 2";
+                        }
+                    }
+                }
+            },
+            defaultTemplatesMap: { "name1": "name2" }
```

Comment:
Now looking at this test I can see that the `defaultTemplatesMap` is not suitable name, because it maps not only default templates. Also I think we need to test the case with a default template too.